lib/code_corps_web/controllers/organization_invite_controller.ex

```diff
@@ -1,34 +1,53 @@
 defmodule CodeCorpsWeb.OrganizationInviteController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2]
```

Comment:
This was no longer in use since you're using `Helpers.Query.id_filter/1` directly in the `show/2`.